lib/ansible/modules/cloud/vmware/vmware_drs_rule.py

```diff
@@ -0,0 +1,288 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2017, Abhijeet Kasurde <akasurde@redhat.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+
+DOCUMENTATION = '''
+---
+module: vmware_drs_rule
+short_description: Configure VMware DRS Affinity rule for virtual machine in given cluster
+description:
+- This module can be used to configure VMware DRS Affinity rule for virtual machine in given cluster.
+version_added: 2.5
+author:
+- "Abhijeet Kasurde (@akasurde)"
+notes:
+- Tested on vSphere 6.5
+requirements:
+- "python >= 2.6"
+- PyVmomi
+options:
+  cluster:
```

Comment:
We should make this more consistent: I've seen `cluster_name` in other modules

Reply:
OK.